ci-operator/jobs/openshift/gcp-pd-csi-driver-operator/openshift-gcp-pd-csi-driver-operator-main-presubmits.yaml

```diff
@@ -568,11 +568,11 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )okd-scos-e2e-aws-ovn,?($|\s.*)
   - agent: kubernetes
-    always_run: false
+    always_run: true
```

Comment:
Why did this change from false to true?

Reply:
This is done by the `make jobs` step. I manually put it back to `false` to see how it goes.

Let's see how the CI goes. If it fails, due to this, I'll have to put it back to `true`.

Reply:
CI is ok with it, so I'll leave it.